RELEASE.md

```diff
@@ -0,0 +1,100 @@
+# How to Update the LSP and its Consumers
+
+## Prerequisites
+
+Make sure you have the following installed:
+
+- A recent release of Rust's `stable` branch
+- `npm`
+- `wasm-pack`
+
+To install `wasm-pack` you can simply run `make install-wasm-pack` from the project root of `flux-lsp`.
+
+You should also have the most recent versions of the following repos pulled down:
+- [ `flux-lsp` ](https://www.github.com/influxdata/flux-lsp)
+- [ `flux-lsp-cli` ](https://www.github.com/influxdata/flux-lsp-cli)
+- [ `flux-lsp` ](https://www.github.com/influxdata/vsflux)
+- [ influxdb ](https://www.github.com/influxdata/influxdb)
+
+## Testing Locally
+
+`cd` into `flux-lsp`.
+
+If you want to test the LSP with a new Flux release, open `Cargo.toml` and look for the `flux` dependency. Replace the `tag` value with whatever the tag for the current release is.
+
+Then run the tests. This should always be done with `make test`, which will run the tests all on a single thread. If you just run `cargo test` you will see random tests passing and failing. An [issue](https://github.com/influxdata/flux-lsp/issues/173) has been created for this, and is currently being addressed by the Flux team. For the time being, just use `make test`.
+
+After confirming that all the tests pass, run `make wasm` to compile with a docker container, or `make wasm-local` to compile locally. This will create two folders: `pkg-node` and `pkg-browser`. Each is a ready-to-publish npm package for different wasm compilation targets.
+
+Navigate to the `vsflux` repo, and open `package.json`. Find the `@influxdata/flux-lsp-node` dependency, and replace the version number with `file:`, followed by the full file path to the `pkg-node` directory in your local copy of `flux-lsp`. It should look something like:
+
+```json
+"dependencies": {
+    "@influxdata/flux-lsp-node": "file: /home/janedoe/projects/flux-lsp/pkg-node"
+}
+```
+
+Then, from the root of `vsflux`, run `npm install`. 
+
+Finally, open up the `vsflux` project in VS Code, click on the `Run` tab in the sidebar, and then click the green arrow at the top of the pane. This should open up a new VS Code window that is running your local version of the extension, rather than the one available on the marketplace. Confirm that any recent changes are working as expected.
```

Comment:
This step probably isn't strictly necessary; the tests in the LSP tests should suffice. I just thought it would be helpful to have the process for this in writing. I can add something that clarifies that this step is an optional sanity check.